src/core/lwip/pbuf.h

```diff
@@ -70,6 +70,7 @@ typedef struct {
         void *mdesc;
         void *opaque;
     };
+    uintptr_t opaque_sq;
```

Comment:
I suggest to rename to soemthing like opaque_tx_userdata or opaque_userdata or opaque_ctx

Reply:
Done. Removed the field completely, because current implementation calls the callback from sockinfo_tcp context, so it has access to the value from the socket object.